From 9d583644ba2273c30b6349b8346fbd224007c8b6 Mon Sep 17 00:00:00 2001
From: Alexander Graf <agraf@suse.de>
Date: Thu, 18 May 2017 15:33:43 +0200
Subject: [PATCH 130/221] arm64: Export arch_setup_dma_ops
Patch-mainline: Not yet, vendor is informed and will push upstream
References: FATE#320030

To build the draa_net driver as module, we need to export the dma ops
setting function to modules. Do so for now until upstream resolved it.

Signed-off-by: Alexander Graf <agraf@suse.de>

---
 arch/arm64/mm/dma-mapping.c | 1 +
 1 file changed, 1 insertion(+)

diff --git a/arch/arm64/mm/dma-mapping.c b/arch/arm64/mm/dma-mapping.c
index af6409f..e50e58c 100644
--- a/arch/arm64/mm/dma-mapping.c
+++ b/arch/arm64/mm/dma-mapping.c
@@ -977,3 +977,4 @@ void arch_setup_dma_ops(struct device *dev, u64 dma_base, u64 size,
 	dev->archdata.dma_coherent = coherent;
 	__iommu_setup_dma_ops(dev, dma_base, size, iommu);
 }
+EXPORT_SYMBOL_GPL(arch_setup_dma_ops);
-- 
1.8.5.6

